.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,21 @@
+name: Build and deploy gh-pages branch with Mkdocs
+
+on:
+  # Runs every time master branch is updated
+  push:
+    branches: ["master"]
+
+jobs:
+  Build-and-Deploy-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: bartgol/scream
+          ref: master
+          fetch-depth: 0 # Needed, or else gh-pages won't be fetched, and push rejected
+      - name: Install python deps
+        run: pip install mkdocs pymdown-extensions mkdocs-material
```

Comment:
Might want to change to `pip3` here, depending on the AT system. It'll be nice when `pip` means "pip for Python 3" at some distant point in the future.

Reply:
How about `python3 -m pip` just to be sure?